tasks/configure.yml

```diff
@@ -163,12 +165,27 @@
   debug: msg={{ ssh_key.stdout_lines }}
   when: archivematica_src_configure_dashboardsettings is defined
 
+- name: "Create archivematica user in AtoM server"
+  user:
+    name: "archivematica"
+    group: "users"
+    system: True
+    home: "/home/archivematica"
+    createhome: True
+    generate_ssh_key: True
```

Comment:
Why is they key created on atom?

Reply:
Thanks Santi, I have just removed the "generate_ssh_key" line. The ".ssh" directory is created by the next task with the `authorized_key` module.